### ISSUE TITLE
treemap - all points are black after drilldown

### ISSUE THREAD

**Issue by sebastianbochan** [highslide-software/highcharts.com#3995]
Example: 
- http://jsfiddle.net/vvz5azbg/1/

Click on the A1, points are black, but when we click on B1 all is fine.

**Comment by jon-a-nygaard** [comment]
Currently working on this.

The bug can be avoided by setting min and max values on the colorAxis:
```javascript
colorAxis: {
  min: 0,
  max: 100,
  minColor: ' #d03b53',
  maxColor: '#23c874'
}
```

**Comment by highslide-software** [comment]
While waiting for our next maintenance release, you can add this line after Highcharts is loaded:

```js
// Drop-in fix for #3995:
Highcharts.seriesTypes.treemap.prototype.getExtremesFromAll = true;
```
[View it live](http://jsfiddle.net/highcharts/vvz5azbg/3/).
